ui/apps/platform/cypress/integration/clusters/clusterDeletion.test.js

```diff
@@ -61,49 +62,49 @@ describe('Cluster page clusterRetentionInfo', () => {
 
             visitClusterById(cluster.id, staticResponseMap);
 
-            assertClusterNameInSidePanel(clusterName);
+            assertClusterNameInHeading(clusterName);
         });
     }
 
     it('should display in 30 days in Cluster Deletion widget for mock request', () => {
         const clusterRetentionInfo = { daysUntilDeletion: 30 };
         visitClusterWithRetentionInfo(clusterRetentionInfo);
 
-        cy.get('div:contains("Cluster Deletion"):contains("Not applicable")');
+        assertClusterDeletionInSummary('in 30 days');
```

Comment:
Incorrect test criterion from copy-paste matched **Not applicable** in the clusters list for clusters side panel but failed for cluster page.